### PR TITLE
feat: vertical discount cards

### PR DESCRIPTION
**Pull request by MariiaMelnychenko**
## Type of Change

- [x] ✨ **New feature** (non-breaking change which adds functionality)
- [ ] 🛠️ **Bug** fix (non-breaking change which fixes an issue)
- [ ] 🧪 **Test** - Adds or updates tests.
- [ ] ❌ **Breaking** change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 **Code refactor**
- [ ] ✅ **Build configuration change**
- [ ] 📝 **Documentation**
- [ ] 🗑️ **Chore**

### CODE REVIEW

File: lib/components/discounts/widget/body/discounts_body_widget.dart [hunk 1]
```diff
@@ -19,10 +19,16 @@ class DiscountsBodyWidget extends StatelessWidget {
   }
 }
 
-class _DiscountsBodyWidget extends StatelessWidget {
+class _DiscountsBodyWidget extends StatefulWidget {
```
Comment (DemianKryk) [inline lib/components/discounts/widget/body/discounts_body_widget.dart:22]: don't change this widget to StatefulWidget!!!

File: lib/components/discounts/widget/body/discounts_body_widget.dart [hunk 2]
```diff
@@ -19,10 +19,16 @@ class DiscountsBodyWidget extends StatelessWidget {
...
   const _DiscountsBodyWidget({required this.scrollController});
   final ScrollController? scrollController;
 
+  @override
+  State<_DiscountsBodyWidget> createState() => _DiscountsBodyWidgetState();
+}
+
+class _DiscountsBodyWidgetState extends State<_DiscountsBodyWidget> {
+  bool _isGridView = false;
```
Comment (DemianKryk) [inline lib/components/discounts/widget/body/discounts_body_widget.dart:31]: I said use cubit for this

File: pubspec.yaml [hunk 3]
```diff
@@ -175,7 +176,7 @@ sentry:
 
   # set by env. var. SENTRY_AUTH_TOKEN
   # We need this variable only when push release build. Used only when upload_source_maps
-  auth_token:
```
Comment (DemianKryk) [inline pubspec.yaml:178]: not change it

**Review by DemianKryk — CHANGES_REQUESTED** [review]
Please, see comments

File: pubspec.yaml (hunk 3 above)
Comment (MariiaMelnychenko) [reply pubspec.yaml:178]: Nothing has changed there

Comment (DemianKryk) [reply pubspec.yaml:178]: You removed ' ', we use this in the Workflow

**Review by DemianKryk — DISMISSED** [review]

**Pull request merged**